develop/api_udf.rst

```diff
@@ -110,6 +110,39 @@ reference table
 
 	SELECT upgrade_to_reference_table('nation');
 
+.. _mark_tables_colocated:
+
+mark_tables_colocated
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+The mark_tables_colocated() function takes a distributed table (the source), and a list of others (the targets), and puts the targets into the same co-location group as the source. If the source is not yet in a group, this function creates one, and assigns the source and targets to it.
```

Comment:
Can we add a few notes:
- That it will error out if the attributes don't match?
- That usually you should use colocate-with option in create_distributed_table(), rather than doing it post-facto with this UDF.

Reply:
Added those notes.